src/screens/Followings/elements/TabBar/FollowingTabBar.js

```diff
@@ -2,8 +2,8 @@ import * as React from 'react';
 import Animated from 'react-native-reanimated';
 import {StyleSheet, TouchableOpacity, View} from 'react-native';
 
-import {colors} from '../../../../utils/colors';
 import {fonts} from '../../../../utils/fonts';
+import {COLORS} from '../../../../utils/theme';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import statement has been updated to use `COLORS` from the `utils/theme` module instead of `colors`. This change is consistent and improves maintainability by using a standardized naming convention for color constants.
```diff
- import {colors} from '../../../../utils/colors';
+ import {COLORS} from '../../../../utils/theme';
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->